disassemblers/ofrak_angr/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include requirements*.txt
```

Comment:
For here and below, should all requirements be included, or just `requirements.txt` (are others needed for pip install)?

Reply:
Yeah, there are other files like `requirements-docs.txt` required for the corresponding "extras" install (`pip install ofrak[docs]`). Besides supporting those install patterns via pip, because of the way the setup.py is run, those files must always be there regardless of whether an extra rule is used, otherwise a file-not-found error is raised.